### ISSUE TITLE
NASA Gauges

### ISSUE THREAD

**Issue by Alexustas** [Alexustas/Props-requests#16]
![bandicam 2015-10-22 03-05-41-593](https://cloud.githubusercontent.com/assets/6713671/10650250/2d389734-786a-11e5-9fb4-3e91d2931576.jpg)
![bandicam 2015-10-22 03-05-28-548](https://cloud.githubusercontent.com/assets/6713671/10650254/304937c6-786a-11e5-9433-61d653c19e32.jpg)
![bandicam 2015-10-22 03-05-18-116](https://cloud.githubusercontent.com/assets/6713671/10650257/3234e4fe-786a-11e5-875c-a10ce4705734.jpg)
![bandicam 2015-10-22 03-06-25-935](https://cloud.githubusercontent.com/assets/6713671/10650258/344afe4a-786a-11e5-9f91-2f45e8f18f9b.jpg)
![bandicam 2015-10-22 03-06-42-829](https://cloud.githubusercontent.com/assets/6713671/10650259/35a5e94e-786a-11e5-8d17-fe8448af2dfb.jpg)
![bandicam 2015-10-22 03-06-51-988](https://cloud.githubusercontent.com/assets/6713671/10650263/36df9f3a-786a-11e5-904b-b70c5a590c9e.jpg)

**Comment by MOARdV** [comment]
It's difficult for me to decide if I like this better, or the VPG_Dual props you made a couple of months ago.

**Comment by Alexustas** [comment]
Design of VPG is mainly inspired by the design of aircraft gauges

![bandicam 2015-10-22 11-52-21-443](https://cloud.githubusercontent.com/assets/6713671/10658180/426a6d52-78b6-11e5-9710-f8426a571350.jpg)

while this gauge is almost a replica of the classic NASA spacecraft instruments

![bandicam 2015-10-22 11-54-02-575](https://cloud.githubusercontent.com/assets/6713671/10658228/9b01c0c8-78b6-11e5-9889-985ff2d7a5ed.jpg)
![bandicam 2015-10-22 11-55-53-690](https://cloud.githubusercontent.com/assets/6713671/10658230/9d10aece-78b6-11e5-9070-002f2f8968b8.jpg)
![bandicam 2015-10-22 12-17-55-299](https://cloud.githubusercontent.com/assets/6713671/10658269/0b2f33d0-78b7-11e5-9ed9-ebd650028650.jpg)

**Comment by MOARdV** [comment]
Yeah, I know the difference.  I still like the look of the aviation gauges.  They add a little bit of color.

**Comment by MOARdV** [comment]
Actually, the more I look at it, the more I like the NASA gauge.  Will there be a single-needle / single gauge variant as well (like the OAMS propellant gauge in Gemini, just left and below of center in the last picture you posted)?

**Comment by Alexustas** [comment]
that's what I have at the moment

https://youtu.be/0Y0YWuNrDy4

it is possible to change the color of scale to black, remove color markup and change the color (dark grey, white, yellow and red)  of the arrows in the config file

**Comment by MOARdV** [comment]
Ah, okay.  Very configurable, then.  That will be nice to have.

**Comment by MOARdV** [comment]
Are these gauges available yet?  Or are they going to be part of an official ASET Historical Props pack?

Also, feedback on the ARRT that you provided a while back: I would recommend changing the RNG RT variable for RNDZMODE (in the SELECT ALTRANGERTVALUE custom variable) to use APPROACHSPEED instead of TRGTSPEED - APPROACHSPEED can be negative, so the rate tape will show when you are moving away from the target.  TRGTSPEED is unsigned, so you can't tell between approach and departure.

**Comment by MOARdV** [comment]
One other thought on the gauges - can there be a texture option to allow them to be a signed value (0 in the middle, + values above, - values below), so it can be used, for instance, as a battery charge rate indicator, trim settings indicator, or control input position repeater?

**Comment by Alexustas** [comment]
Also, feedback on the ARRT that you provided a while back: I would recommend changing the RNG 

> RT variable for RNDZMODE (in the SELECT ALTRANGERTVALUE custom variable) to use >APPROACHSPEED instead of TRGTSPEED - APPROACHSPEED can be negative, so the rate tape >will show when you are moving away from the target. TRGTSPEED is unsigned, so you can't tell >between approach and departure.

Approach speed now showing on the DSKY when 'APPR' mode is selected. as well as "APPROACH" and "DEPARTURE" lights

**Comment by Alexustas** [comment]
> One other thought on the gauges - can there be a texture option to allow them to be a signed value >(0 in the middle, + values above, - values below), so it can be used, for instance, as a battery charge >rate indicator, trim settings indicator, or control input position repeater?

Yes, such a device is also in my plans, but it is not ready yet. That's why I do not publish them

**Comment by Alexustas** [comment]
here are a few new props
![screenshot101](https://cloud.githubusercontent.com/assets/6713671/15059214/db3e25c4-1342-11e6-95c7-9b0913192b85.png)
![screenshot97](https://cloud.githubusercontent.com/assets/6713671/15059221/e70fb3fe-1342-11e6-9a50-de6983ef1972.png)
![screenshot96](https://cloud.githubusercontent.com/assets/6713671/15059224/ea674a4e-1342-11e6-8b3f-fac7b8b62d96.png)
![Uploading screenshot103.png…]()

**Comment by DeputyLOL** [comment]
I'm really interested in that 'impact speed' dial, has this been released yet?
I think I could modify it to make a 'cabin altitude' gauge. :D

**Comment by Alexustas** [comment]
> I'm really interested in that 'impact speed' dial,

which altitude range you need?

**Comment by Alexustas** [comment]
and how this device should work in KSP?

**Comment by DeputyLOL** [comment]
I'm sure that a gauge that looked like this could have multiple uses. 
![image](https://cloud.githubusercontent.com/assets/12616126/15211470/49d2e6ec-1833-11e6-8722-78e0b2b56fed.png)
It doesn't have to be cabin altitud, i could then modify the texture if needed. This is your analogue speed gauge that i use for cabin altitude:
![image](https://cloud.githubusercontent.com/assets/12616126/15211572/e10bc5c4-1833-11e6-8543-47ea83c22fd0.png)

**Comment by DeputyLOL** [comment]
Do you want me to start a new request for steam gauges?

**Comment by Alexustas** [comment]
> Do you want me to start a new request for steam gauges?

steam gauges? 0_0

**Comment by DeputyLOL** [comment]
Sorry. Steam = Analogue. Basically like the stuff in the Classic Instruments request.
I can add it to there if you want.

**Comment by Alexustas** [comment]
> I can add it to there if you want.

if it requires new models, you have to create a new request
